marklogic-data-hub-central/ui/e2e/README.md

```diff
@@ -4,28 +4,39 @@ This hc-qa-project is intended to run HC e2e test.
 
 Run the setup script first from under `marklogic-data-hub-central/ui/e2e` directory, to initialize, deploy and run flows:
 
-    ./setup.sh dhs=<true/false> mlHost=<curatoinEndpoint/localhost>
- 
+    For DHS -
+        * Create users as admin using the payload in cypress/fixtures/users for the tests
+        * ./setup.sh dhs=true mlHost=<curationEndpoint>
+            * example of curation endpoint is dwgz7dnkj.ec2dentifier10.a.marklogicsvc.com
```

Comment:
Does "curation endpoint" mean "a host with one or more app servers assigned to the Curation group in ML"? I ask because I don't believe that group is guaranteed to exist in DHS. I think what matters is that the host has all the app servers that setup.sh depends on.

Reply:
curation endpoint is simply the Curation stack: https://docs.marklogic.com/cloudservices/aws/refs/endpoints-and-port-numbers.html and the naming(curation endpoint) is based on its usage across cloud services and our team. The portal API we use to bring up a stack, throws an output saying curationEndpoint=<....> and I understand now from @SameeraPriyathamTadikonda that its now been changed to httpsendpoint.
Its just how curation stack is referred to.

Reply:
I believe curation endpoint is the loadbalancer endpoint which will redirect to MLHost in the cluster.